modules/installation-azure-trusted-launch.adoc

```diff
@@ -19,21 +19,48 @@ include::snippets/technology-preview.adoc[]
 
 .Procedure
 
-* Use a text editor to edit the `install-config.yaml` file prior to deploying your cluster and add the following stanza:
+* Edit the `install-config.yaml` file before deploying your cluster:
+
+** To enable trusted launch only on control plane, add the following stanza:
 +
 [source,yaml]
 ----
-controlPlane: <1>
+controlPlane:
   platform:
     azure:
       settings:
-        securityType: TrustedLaunch <2>
+        securityType: TrustedLaunch
         trustedLaunch:
           uefiSettings:
-            secureBoot: Enabled <3>
-            virtualizedTrustedPlatformModule: Enabled <4>
+            secureBoot: Enabled
+            virtualizedTrustedPlatformModule: Enabled
+----
+
+** To enable trusted launch only on compute node, add the following stanza:
++
+[source,yaml]
+----
+compute:
+  platform:
+    azure:
+      settings:
+        securityType: TrustedLaunch
+        trustedLaunch:
+          uefiSettings:
+            secureBoot: Enabled
+            virtualizedTrustedPlatformModule: Enabled
+----
+
+**  To enable trusted launch on all nodes, add the following stanza:
++
+[source,yaml]
 ----
-<1> Specify `controlPlane.platform.azure` or `compute.platform.azure` to enable trusted launch on only control plane or compute nodes respectively. Specify `platform.azure.defaultMachinePlatform` to enable trusted launch on all nodes.
-<2> Enable trusted launch features.
-<3> Enable secure boot. For more information, see the Azure documentation about link:https://learn.microsoft.com/en-us/azure/virtual-machines/trusted-launch#secure-boot[secure boot].
-<4> Enable the virtualized Trusted Platform Module. For more information, see the Azure documentation about link:https://learn.microsoft.com/en-us/windows/security/hardware-security/tpm/trusted-platform-module-overview[virtualized Trusted Platform Modules].
\ No newline at end of file
+platform:
+  azure:
+    settings:
+      securityType: TrustedLaunch <1>
```

Comment:
Here <1>/<2>/<3> seems no explanation.
The explanations are deleted by accident?

Reply:
one more question: any reason to only adjust section "Enabling trusted launch for Azure VMs"? Section "Enabling confidential VMs" should have similar structure.

Reply:
@jinyunma I have removed  <1>/<2>/<3> . Earlier the explanations were there because we were clubbing everything into a single example. Now that it's split into 3, I think we wouldn't need that anymore.

Reply:
and I agree about modifying "Enabling confidential VMs". I was planning to create a separate PR. In case it's easier for you to review in this PR, I can do that